src/Pages/Model/PageEdit.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Pages\Model;
+
+class PageEdit
```

Comment:
I would create only one Page class model. This class is rather request argument so it shouldn't be in the domain model. This is a command and it's better to move this to `App\Pages\Command` namespace.